src/init/features/frameworks/index.ts

```diff
@@ -57,22 +52,19 @@ export async function doSetup(setup: any, projectId: string): Promise<void> {
 
   utils.logSuccess(`Region set to ${setup.frameworks.region}.`);
 
-  logger.info(clc.bold(`\n${clc.white("===")} Deploy Setup`));
-
-  await promptOnce(
-    {
-      name: "deployMethod",
-      type: "list",
-      default: DEFAULT_DEPLOY_METHOD,
-      message: "How do you want to deploy",
-      choices: ALLOWED_DEPLOY_METHODS,
-    },
-    setup.frameworks
-  );
-
   const backend: Backend | undefined = await getOrCreateBackend(projectId, setup);
   if (backend) {
-    utils.logSuccess(`Successfully created a backend: ${backend.name}`);
+    logger.info();
```

Comment:
what is the purpose of these blank logger.info() lines? just to add blank space?

Reply:
yes, the print statements are cluttered if these are not present. 
Found usage of these in other places of codebase.